hms-app/src/pages/YourIdeas.tsx

```diff
@@ -21,75 +22,87 @@ function YourIdeas() {
     ideas: [],
   } as Hackathon)
 
-  const filteredIdeas = relevantIdeaList.filter((item) => {
-    return item.owner?.id.includes(userId)
+  const filteredIdeas = relevantIdeas.filter((item) => {
+    return item.owner?.user.id.includes(userID)
   })
 
-  const getHackathonID = (hackthonID: string) => {
+  const getParticipant = hackathons.participants?.filter((participant) => {
+    return participant.user.id.includes(userID)
+  })
+
+  const getID = getParticipant?.map((participant) => {
+    return participant.id
+  })
+
+  const setHackathonID = (hackthonID: string) => {
     setSelectedHackweek(hackthonID)
   }
 
-  const getRelevantIdeaList = (relevantIdeaList: Idea[]) => {
-    setRelevantIdeaList(relevantIdeaList)
+  const setRelevantIdeaList = (relevantIdeaList: Idea[]) => {
+    setRelevantIdeas(relevantIdeaList)
   }
 
-  const getHackathonData = (hackathonData: Hackathon) => {
-    setHackathonData(hackathonData)
+  const setHackathonData = (hackathonData: Hackathon) => {
+    setHackathons(hackathonData)
   }
 
-  const getIsLoading = (isLoading: boolean) => {
+  const setThisIsLoading = (isLoading: boolean) => {
     setIsLoading(isLoading)
   }
 
+  useEffect(() => {
+    // @ts-ignore
```

Comment:
Probably a good idea would be to not do this 😁

Reply:
Also, this error that was suppressed here told what I've described above right away 😅 

> Argument of type 'string[] | undefined' is not assignable to parameter of type 'SetStateAction<string>'.
  Type 'undefined' is not assignable to type 'SetStateAction<string>'.